engine/language_client_typescript/artifacts/native.d.ts

```diff
@@ -0,0 +1,133 @@
+/* auto-generated by NAPI-RS */
```

Comment:
Looks like lots of files under `artifacts` get checked in. Did we mean to exclude them from VCS?

Reply:
Ah ha, yes we can delete these now. It gets auto generated during build time. This allows us to do debugging locally a bit easier